CONTRIBUTING.md

```diff
@@ -1,22 +1,79 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Contributing to the Ferdium Website
 
-## Getting Started
+:tada: First off, thanks for taking the time and your effort to make Ferdium better! :tada:
+
+## Table of contents
+
+<!-- TOC depthFrom:2 depthTo:2 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Table of contents](#table-of-contents)
+- [Code of Conduct](#code-of-conduct)
+- [How do I contribute?](#how-do-i-contribute)
+	- [Setup your development environment](#setup-your-development-environment)
+		- [Node.js, pnpm](#nodejs-pnpm)
+		- [Git](#git)
+	- [Clone the repository](#clone-the-repository)
+	- [Install dependencies](#install-dependencies)
+	- [Add your modifications](#add-your-modifications)
+	- [Build the website locally](#build-the-website-locally)
+- [Learn More](#learn-more)
+
+<!-- /TOC -->
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Ferdium Code of Conduct](https://raw.githubusercontent.com/ferdium/ferdium-app/develop/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
```

Comment:
Should we point to the code of conduct from the other repo, or duplicate that file in this one?

Reply:
See below